canonical/store-components
# made social links configurable

## Done
Made social links configurable in the first column

## QA steps
- Go to https://store-components-18.demos.haus/?path=/story/footer--default-story
- Check if there are social link icons in the first column

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-1614
